PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -211,4 +214,17 @@ integration, telemetry connectivity, and Azure Toolkit integration.
       batch request.
 - **Severity: WARNING**
 - **Recommendation**: Use Batch Operations: If the SDK provides a batch operation API, use it to perform multiple
-  actions in a single request.
\ No newline at end of file
+  actions in a single request.
+
+11. #### Authenticating a Non-Azure OpenAI Client with KeyCredential
+
+- **Anti-pattern**: Assigning the endpoint value when creating a Non-Azure OpenAI client using the KeyCredential in
+  .credential(KeyCredential).
+- **Issue**: KeyCredential is the only required parameter in `.credential(KeyCredential)` for authenticating requests to
+  non-Azure OpenAI APIs.
+- **Severity: WARNING**
+- **Recommendation**: Omit Endpoint: Do not specify the endpoint parameter unless you are working with Azure-specific
```

Comment:
**Recommendation**: Omit Endpoint: ?

Not sure I understand the recommendation.